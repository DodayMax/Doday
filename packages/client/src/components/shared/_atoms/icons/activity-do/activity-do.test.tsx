import * as React from 'react';
import { shallow } from 'enzyme';
import { Icons } from '@shared';

describe('ActivityDoType', () => {
  it('renders correctly without props', () => {
    const icon = shallow(<Icons.ActivityDoType />);

    expect(icon).toMatchSnapshot();
  });
});
