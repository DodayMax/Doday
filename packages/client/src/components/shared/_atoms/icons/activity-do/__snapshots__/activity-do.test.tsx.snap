// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActivityDoType renders correctly without props 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <g
        fill="black"
        fillRule="evenodd"
      >
        <path
          d="M10.725 13.472L16.197 8 18 9.803l-7.275 7.274L6 12.352l1.803-1.803z"
          fillRule="nonzero"
        />
        <path
          d="M17.71 4.749a3.622 3.622 0 0 1 3.612 3.63v7.144a3.622 3.622 0 0 1-3.612 3.63H6.515a3.622 3.622 0 0 1-3.612-3.63V8.379a3.622 3.622 0 0 1 3.612-3.63h11.2-.005zm.005-1.543h-11.2c-2.841.003-5.144 2.317-5.147 5.173v7.144c.006 2.854 2.308 5.166 5.147 5.168h11.2c2.841-.002 5.144-2.317 5.147-5.173V8.374c-.006-2.856-2.31-5.168-5.152-5.168h.005z"
        />
      </g>,
      "height": 20,
      "viewBox": "0 0 24 24",
      "width": 20,
      "xmlns": "http://www.w3.org/2000/svg",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <path
            d="M10.725 13.472L16.197 8 18 9.803l-7.275 7.274L6 12.352l1.803-1.803z"
            fillRule="nonzero"
          />,
          <path
            d="M17.71 4.749a3.622 3.622 0 0 1 3.612 3.63v7.144a3.622 3.622 0 0 1-3.612 3.63H6.515a3.622 3.622 0 0 1-3.612-3.63V8.379a3.622 3.622 0 0 1 3.612-3.63h11.2-.005zm.005-1.543h-11.2c-2.841.003-5.144 2.317-5.147 5.173v7.144c.006 2.854 2.308 5.166 5.147 5.168h11.2c2.841-.002 5.144-2.317 5.147-5.173V8.374c-.006-2.856-2.31-5.168-5.152-5.168h.005z"
          />,
        ],
        "fill": "black",
        "fillRule": "evenodd",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "d": "M10.725 13.472L16.197 8 18 9.803l-7.275 7.274L6 12.352l1.803-1.803z",
            "fillRule": "nonzero",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "d": "M17.71 4.749a3.622 3.622 0 0 1 3.612 3.63v7.144a3.622 3.622 0 0 1-3.612 3.63H6.515a3.622 3.622 0 0 1-3.612-3.63V8.379a3.622 3.622 0 0 1 3.612-3.63h11.2-.005zm.005-1.543h-11.2c-2.841.003-5.144 2.317-5.147 5.173v7.144c.006 2.854 2.308 5.166 5.147 5.168h11.2c2.841-.002 5.144-2.317 5.147-5.173V8.374c-.006-2.856-2.31-5.168-5.152-5.168h.005z",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
      ],
      "type": "g",
    },
    "type": "svg",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <g
          fill="black"
          fillRule="evenodd"
        >
          <path
            d="M10.725 13.472L16.197 8 18 9.803l-7.275 7.274L6 12.352l1.803-1.803z"
            fillRule="nonzero"
          />
          <path
            d="M17.71 4.749a3.622 3.622 0 0 1 3.612 3.63v7.144a3.622 3.622 0 0 1-3.612 3.63H6.515a3.622 3.622 0 0 1-3.612-3.63V8.379a3.622 3.622 0 0 1 3.612-3.63h11.2-.005zm.005-1.543h-11.2c-2.841.003-5.144 2.317-5.147 5.173v7.144c.006 2.854 2.308 5.166 5.147 5.168h11.2c2.841-.002 5.144-2.317 5.147-5.173V8.374c-.006-2.856-2.31-5.168-5.152-5.168h.005z"
          />
        </g>,
        "height": 20,
        "viewBox": "0 0 24 24",
        "width": 20,
        "xmlns": "http://www.w3.org/2000/svg",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <path
              d="M10.725 13.472L16.197 8 18 9.803l-7.275 7.274L6 12.352l1.803-1.803z"
              fillRule="nonzero"
            />,
            <path
              d="M17.71 4.749a3.622 3.622 0 0 1 3.612 3.63v7.144a3.622 3.622 0 0 1-3.612 3.63H6.515a3.622 3.622 0 0 1-3.612-3.63V8.379a3.622 3.622 0 0 1 3.612-3.63h11.2-.005zm.005-1.543h-11.2c-2.841.003-5.144 2.317-5.147 5.173v7.144c.006 2.854 2.308 5.166 5.147 5.168h11.2c2.841-.002 5.144-2.317 5.147-5.173V8.374c-.006-2.856-2.31-5.168-5.152-5.168h.005z"
            />,
          ],
          "fill": "black",
          "fillRule": "evenodd",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "d": "M10.725 13.472L16.197 8 18 9.803l-7.275 7.274L6 12.352l1.803-1.803z",
              "fillRule": "nonzero",
            },
            "ref": null,
            "rendered": null,
            "type": "path",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "d": "M17.71 4.749a3.622 3.622 0 0 1 3.612 3.63v7.144a3.622 3.622 0 0 1-3.612 3.63H6.515a3.622 3.622 0 0 1-3.612-3.63V8.379a3.622 3.622 0 0 1 3.612-3.63h11.2-.005zm.005-1.543h-11.2c-2.841.003-5.144 2.317-5.147 5.173v7.144c.006 2.854 2.308 5.166 5.147 5.168h11.2c2.841-.002 5.144-2.317 5.147-5.173V8.374c-.006-2.856-2.31-5.168-5.152-5.168h.005z",
            },
            "ref": null,
            "rendered": null,
            "type": "path",
          },
        ],
        "type": "g",
      },
      "type": "svg",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
